docs/tutorials/timeseries/index.md

```diff
@@ -6,7 +6,7 @@ and does not require you to manually deal with cumbersome issues like model sele
 
 Under the hood, AutoGluon combines various state of the art forecasting algorithms.
 These include established statical methods like ETS and ARIMA from
-[`StatsForecast`](https://github.com/Nixtla/statsforecast) and [`statsmodels`](https://statsmodels.org/) libraries,
+[`StatsForecast`](https://github.com/Nixtla/statsforecast),
```

Comment:
My apologies for this question. Is there anywhere in AGTS documented what are the models available to the user?

Reply:
Ah, it is here!
http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-3513/426f107/tutorials/timeseries/forecasting-model-zoo.html

Reply:
Thanks, that's a really good point! I will remove the references from Model Zoo as well.

Reply:
Actually, it seems that the references to the models got updated automatically and there is nothing to change there 😄

Reply:
Yes, it is interesting :)